packages/experiments/src/components/UnifiedPicker/UnifiedPeoplePicker/UnifiedPeoplePicker.tsx

```diff
@@ -11,43 +10,29 @@ import {
 import { UnifiedPicker } from '../UnifiedPicker';
 
 export const UnifiedPeoplePicker = (props: IUnifiedPeoplePickerProps): JSX.Element => {
-  // update the suggestion like componentWillReceiveProps
```

Comment:
From looking at this code, `peopleSuggestions` and `peopleSelectedItems` were **only** being used to trigger generation of new render callbacks. The same thing can be achieved in a simpler way by just making the callbacks directly dependent on those props.